docs/tutorial/index.md

```diff
@@ -13,62 +13,36 @@ tags:
 search:
   boost: 2
 ---
-# Tutorial Overview
+# Tutorial: Master workflow automations
 
-This tutorial provides a guided walk-through of Prefect core concepts and instructions on how to use them.
+This tutorial helps you get started with Prefect and become familiar with developing, deploying, and managing Prefect workflows.
 
-By the end of this tutorial you will have:
+After you build your deployment, you can read other [guides](/guides) to learn how to integrate other services and add more features to your workflows.
 
-1. [Created a flow](/tutorial/flows/)
-2. [Added tasks to it](/tutorial/tasks/)
-3. [Deployed and run the flow locally](/tutorial/deployments/)
-4. [Created a work pool and run the flow remotely](/tutorial/work-pools/)
+## Before you begin
 
-These four topics will get most users to their first production deployment.
+Complete the following steps to start the tutorial:
 
-Advanced users that need more governance and control of their workflow infrastructure can go one step further by:
+1. Install [Python](https://www.python.org/downloads/).
+1. Install Prefect:
```

Comment:
```suggestion
1. Install Prefect:
<div class="terminal">
```

Reply:
Wrapping in a `div` tag for additional styling that we use with bash commands and removing the indent. Trying to keep the format consistent.